vcd/config_test.go

```diff
@@ -155,6 +155,9 @@ type TestConfig struct {
 		UploadProgress  bool   `json:"uploadProgress,omitempty"`
 		MediaName       string `json:"mediaName,omitempty"`
 	} `json:"media"`
+	Misc struct {
+		LdapContainer string `json:"ldapContainer,omitempty"`
```

Comment:
Would be nice to have a comment explaining when this should be used.

Reply:
I clearly put wrong explanation for this. So the problem is we use a public docker container `rroemhild/test-openldap` with LDAP server to run a few tests and make sure that LDAP works.

Recently docker registry introduced throttling and sometimes we hit this limit, then our tests fails because image cannot be pulled. This parameter allows to pull the same LDAP container (because tests are wired to use it) from another mirror registry. In fact I missed adding details into `sample_vcd_test_config.json` initially. I added this comment there, because that is where we try to document. 

I can also add it to the struct, but we have not did that before so a single comment here might look odd.